data/words/famous_inspirational-quotes.json

```
@@ -0,0 +1,1311 @@
+           "from":"William B. Sprague"
+        },
+        {  
+           "text":"It’s easier to ask forgiveness than it is to get permission.",
+           "from":"Grace Hopper"
+        },
+        {  
+           "text":"Twenty years from now you will be more disappointed by the things that you didn’t do than by the ones you did do.",
+           "from":"Mark Twain"
+        },
+        {  
+           "text":"One day your life will flash before your eyes. Make sure it is worth watching.",
+           "from":"Mooie"
+        },
+        {  
+           "text":"“I think it’s wrong that only one company makes the game Monopoly.",
+           "from":"Steven Wright"
+        },
+        {  
+           "text":"Ever notice how it’s a penny for your thoughts, yet you put in your two-cents? Someone is making a penny on the deal.",
+           "from":"Steven Wright"
+        },
+        {  
+           "text":"Catch a man a fish, and you can sell it to him. Teach a man to fish, and you ruin a wonderful business opportunity.",
+           "from":"Karl (maybe Groucho) Marx"
+        },
+        {  
+           "text":"I used to sell furniture for a living. The trouble was, it was my own.",
+           "from":"Les Dawson"
+        },
+        {  
+           "text":"Marking dynamos for repair $10,000.00—2 hours labor $10.00; knowing where to mark $9,990.00.",
+           "from":"Invoice from Charles Steinmetz"
+        },
+        {  
+           "text":"By working faithfully eight hours a day you may eventually get to be boss and work twelve hours a day.",
+           "from":"Robert Frost"
+        },
+        {  
+           "text":"My son is now an ‘entrepreneur’. That’s what you’re called when you don’t have a job.",
+           "from":"Ted Turner"
+        },
+        {  
+           "text":"I didn’t go to college, but if I did, I would’ve taken all my tests at a restaurant, ’cause ‘The customer is always right.’",
+           "from":"Mitch Hedberg"
+        },
+        {  
+           "text":"Formal education will make you a living. Self education will make you a fortune.",
+           "from":"Jim Rohn"
+        },
+        {  
+           "text":"The greatest reward in becoming a millionaire is not the amount of money that you earn. It is the kind of person that you have to become to become a millionaire in the first place.",
+           "from":"Jim Rohn"
+        },
+        {  
+           "text":"If you’re not learning while you’re earning, you’re cheating yourself out of the better portion of your compensation.",
+           "from":"Napoleon Hill"
+        },
+        {  
+           "text":"A business that makes nothing but money is a poor business.",
+           "from":"Henry Ford"
+        },
+        {  
+           "text":"After a certain point, money is meaningless. It ceases to be the goal. The game is what counts.",
+           "from":"Aristotle"
```

This appears to be attributed to Aristotle Onassis (1906-1975) rather than Aristotle (384 BC-322 BC):
```suggestion
           "from":"Aristotle Onassis"
```